map/map_height_fitter/config/map_height_fitter.param.yaml

```diff
@@ -0,0 +1,3 @@
+/**:
+  ros__parameters:
+    map_loader_name: /map/pointcloud_map_loader
```

Comment:
@TaikiYamada4 Keeping it consistent to other param.yaml files, could you make the value double-quoted?

Reply:
Indeed. I fixed it now. Thank you for the comment!